utils/i2c-tools/Makefile

```diff
@@ -56,7 +52,6 @@ define Package/python3-smbus
   CATEGORY:=Languages
   TITLE:=Python bindings for the SMBUS
   DEPENDS:=+libi2c +python3-light
-  VARIANT:=python3
```

Comment:
i'd poke the maintainer of i2c-tools here a bit;
some variant mechanism is a slightly outside of python scope here;

Reply:
It looks like variants were added in 12caa41dd1c4f7b931c53e39e0b539c6fc627fee so that it can use the Python build recipes for the Python packages. I think adding the `bin` variant was so that the `bin` packages can have a different `Build/Compile`.

Still, probably a good idea to ping @dangowrt (and @Cynerd) :joy:

Side note: I've found that if some packages in a Makefile have a variant and other packages (in the same Makefile) don't (like newt or nmap before this change), the `BUILD_VARIANT` value may be set for the non-variant packages during build. (At least, they all share the same build_dir with the variant name in the build_dir name.) It's probably not safe to mix variant and non-variant packages in the same Makefile.